config/rbac/namespaced/role.yaml

```diff
@@ -0,0 +1,168 @@
+
+---
```

Comment:
nit: remove

Reply:
It's generated this way, believe it or not. (I believe it)